TelCo.ColorCoder/mainProgram.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Drawing;
+using System.Text;
+
+namespace ConsoleApp1
+{
+    class mainProgram
+    {
+        private static void Main(string[] args)
+        {
+
+
+            colorManual.manual();
+            int pairNumber = 4;
+            ColorPair testPair1 = Program.GetColorFromPairNumber(pairNumber);
+           
+
+           
+            pairNumber = 5;
+            testPair1 = Program.GetColorFromPairNumber(pairNumber);
+            Console.WriteLine("[In]Pair Number: {0},[Out] Colors: {1}\n", pairNumber, testPair1);
+            Debug.Assert(testPair1.majorColor == Color.White);
+            Debug.Assert(testPair1.minorColor == Color.SlateGray);
+
+            pairNumber = 23;
+            testPair1 = Program.GetColorFromPairNumber(pairNumber);
+            Console.WriteLine("[In]Pair Number: {0},[Out] Colors: {1}\n", pairNumber, testPair1);
+            Debug.Assert(testPair1.majorColor == Color.Violet);
+            Debug.Assert(testPair1.minorColor == Color.Green);
+
+            ColorPair testPair2 = new ColorPair() { majorColor = Color.Yellow, minorColor = Color.Green };
+            pairNumber = ColorPair.GetPairNumberFromColor(testPair2);
+            Console.WriteLine("[In]Colors: {0}, [Out] PairNumber: {1}\n", testPair2, pairNumber);
+            Debug.Assert(pairNumber == 18);
+
```

Comment:
Try writing a test for the `colorManual` class

Reply:
sure will do.